compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/intermediate/PropertyLValue.kt

```diff
@@ -144,7 +146,8 @@ class AccessorPropertyLValue(
     }
 
     override fun load(): IrExpression =
-        callReceiver.adjustForCallee(getterDescriptor!!).call { dispatchReceiverValue, extensionReceiverValue ->
+        // TODO: Use context receiver values
```

Comment:
Add tests for:
- reading propertu with context receivers (`foo().a`);
- assigning property with context receivers (`foo().a = b`);
- safe assignment for a property with context receivers (`foo()?.a = b` - yes, it's legal in Kotlin);
- compound assignment for a property with context receiver (`foo().a += b`).

Reply:
Please, add a box test that would involve explicit receiver expression with some side effect (`foo().a = b` above).

Reply:
Thanks, found a TODO in safe calls generation. [Commit with the test.](https://github.com/JetBrains/kotlin/pull/4488/commits/ead6d37aee92e05ba8b969241b1a63db034cc4b8)